pkg/cmd/cli/cmd/rollback.go

```diff
@@ -306,7 +300,7 @@ func (o *RollbackOptions) findResource(targetName string) (runtime.Object, error
 // the deployment matching desiredVersion will be returned. If desiredVersion
 // is <=0, the last completed deployment which is older than the config's
 // version will be returned.
-func (o *RollbackOptions) findTargetDeployment(config *deployapi.DeploymentConfig, desiredVersion int) (*kapi.ReplicationController, error) {
+func (o RollbackOptions) findTargetDeployment(config *deployapi.DeploymentConfig, desiredVersion int) (*kapi.ReplicationController, error) {
```

Comment:
Makes sense to only have changes from Complete propagate into the Options
struct

On Fri, Oct 16, 2015 at 4:12 PM, Dan Mace <notifications@github.com> wrote:

> In pkg/cmd/cli/cmd/rollback.go
> <https://github.com/openshift/origin/pull/5157#discussion_r42246343>:
>
> > @@ -306,7 +300,7 @@ func (o *RollbackOptions) findResource(targetName string) (runtime.Object, error
> >  // the deployment matching desiredVersion will be returned. If desiredVersion
> >  // is <=0, the last completed deployment which is older than the config's
> >  // version will be returned.
> > -func (o *RollbackOptions) findTargetDeployment(config *deployapi.DeploymentConfig, desiredVersion int) (*kapi.ReplicationController, error) {
> > +func (o RollbackOptions) findTargetDeployment(config *deployapi.DeploymentConfig, desiredVersion int) (*kapi.ReplicationController, error) {
>
> Why aren't you using pointer receivers?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5157/files#r42246343>.
>